crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -206,8 +206,7 @@ impl BuildSpecCommand {
 			#[cfg(not(test))]
 			sleep(Duration::from_secs(3))
 		}
-
-		cli.info("Generating chain specification...")?;
+		let multi = multi_progress("Generating chain specification...");
```

Comment:
Not sure multi has much value here - there are no actions happening in parallel which need to be shown at the same time and the output in the spinner is replaced in any case. Would simplify the code to just use a single spinner.

Reply:
Solved! It looks nicer now with the `spinner`. I had to include a `spinner.clear();` in case the node has to be built otherwise output in the terminal very messy as Ale indicated here https://github.com/r0gue-io/pop-cli/pull/334 (That was the reason to use `multi_progress`).